hadoop-ozone/dist/src/main/smoketest/topology/cli.robot

```diff
@@ -22,18 +22,27 @@ Test Timeout        5 minutes
 
 *** Variables ***
 
-
 *** Test Cases ***
 Run printTopology
     ${output} =         Execute          ozone admin printTopology
-                        Should contain   ${output}         10.5.0.7(ozone-topology_datanode_4_1.ozone-topology_net)    IN_SERVICE    /rack2
+                        Should Contain   ${output}         State = HEALTHY
+                        Should Contain          ${output}         IN_SERVICE
+                        Should Match Regexp     ${output}         .*datanode[-_]\\d+.*IN_SERVICE.*
+
 Run printTopology -o
     ${output} =         Execute          ozone admin printTopology -o
-                        Should contain   ${output}         Location: /rack2
-                        Should contain   ${output}         10.5.0.7(ozone-topology_datanode_4_1.ozone-topology_net) IN_SERVICE
+                        Should Contain   ${output}         State = HEALTHY
+                        Should Contain   ${output}         Location:
+                        Should Match Regexp   ${output}         .*ozone.*datanode[-_]\\d+.*IN_SERVICE.*
+
 Run printTopology --operational-state IN_SERVICE
     ${output} =         Execute          ozone admin printTopology --operational-state IN_SERVICE
-                        Should contain   ${output}         10.5.0.7(ozone-topology_datanode_4_1.ozone-topology_net)    IN_SERVICE    /rack2
+                        Should Contain   ${output}         State = HEALTHY
+                        Should Contain   ${output}         IN_SERVICE
+                        Should Match Regexp   ${output}         .*ozone.*datanode[-_]\\d+.*IN_SERVICE.*
```

Comment:
Let's avoid duplicating assertions.  Please extract a `Keyword` and use it in all test cases.

Reply:
Done